README.md

```diff
@@ -1,2 +1,71 @@
 # uphold/litecoind
 A litecoind docker image.
+
+[![uphold/litecoind][docker-pulls-image]][docker-hub-url] [![uphold/litecoind][docker-stars-image]][docker-hub-url] [![uphold/litecoind][docker-size-image]][docker-hub-url] [![uphold/litecoind][docker-layers-image]][docker-hub-url]
+
+## Supported tags and respective `Dockerfile` links
+- `0.10.4.0-alpine`, `0.10.4-alpine` ([0.10.4/alpine/Dockerfile](https://github.com/uphold/docker-litecoin/blob/master/0.10.4/alpine/Dockerfile))
+
+## What is litecoind?
+_from [litecoinwiki](https://litecoin.info/Litecoin)_
```

Comment:
Suggestion: remove this line and link to the Litecoin wiki in the next sentence.